SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -142,23 +142,28 @@ bool CreateGlobalStateManager(Worker_SnapshotOutputStream* OutputStream)
 	return Worker_SnapshotOutputStream_WriteEntity(OutputStream, &GSM) != 0;
 }
 
-bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
+bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream, Worker_EntityId& NextAvailableEntityID)
 {
 	// Set up grid of "placeholder" entities to allow workers to be authoritative over _something_.
-	int PlaceholderCount = SpatialConstants::PLACEHOLDER_ENTITY_ID_LAST - SpatialConstants::PLACEHOLDER_ENTITY_ID_FIRST + 1;
```

Comment:
We should update the SpatialConstants.h to no longer have `PLACEHOLDER_ENTITY_ID_LAST`

Reply:
I think it's updated - take a look at the diff above.